### PR TITLE
Rebuild v11.8.5 to include missing files

### PR DESCRIPTION
**Pull request by leofang**
xref: https://github.com/NVIDIA/cuda-python/pull/231

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
Would it make sense to add a test for this?

Perhaps something simple like...?

```cython
from cuda.ccudart cimport cudaStream_t

def func():
    cdef cudaStream_t s = <cudaStream_t>NULL
```

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Review by bdice — APPROVED** [review]

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -51,6 +53,18 @@ requirements:
     - pywin32  # [win64]
 
 test:
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (leofang) [inline recipe/meta.yaml:58]: Unlike in the CUDA 12 case, where the C/C++ compilers are brought in via `cuda-nvcc`, for CUDA 11 we need to explicitly request a host compiler here.

I also noticed that if we don't do this, on Windows the MSVC from the VM would be invoked.

Comment (jakirkham) [reply recipe/meta.yaml:58]: We can just skip the test on Windows CUDA 11 for now and follow up separately

**Pull request merged**